### ISSUE TITLE
No validation of dcterms:conformsTo metadata against EPUB Accessibility 1.1 specification

### ISSUE THREAD

**Issue by man-we** [daisy/ace#443]
It appears that Ace 1.3.7 does not validate the metadata entered in dcterms:conformsTo against the current list of allowed conformance strings, according to [https://www.w3.org/TR/epub-a11y-11/#sec-conf-reporting-pub](url)

An EPUB containing this purely fictional conformance statement did not raise any kind of error in Ace 1.3.7 when I tested it:

`<meta property="dcterms:conformsTo">EPUB Accessibility 0815 - WCAG 17.6 Level XYZ</meta>`

(For what it's worth, neither did epubcheck 5.2.1)

Is this check consciously omitted in Ace for a reason that I failed to see?

**Comment by clapierre** [comment]
There could be multiple conformance statements, I know Macmillan Learning has their own set of conformance statement so we can't flag anything that is not part of our allowed list.
However, I think if it doesn't find one that matches, some kind of warning or information about unrecognized conformance claims could be useful.

**Comment by mattgarrish** [comment]
I think it would be useful to see if we can generate a warning here. No one should be using the same name for their own standard with only slight modifications. We could even make that a rule in the specification that proprietary identifiers must not start with "EPUB Accessibility", if need be.

In that case, it should be possible to tokenize the string on whitespace and if the first two segments match "EPUB" and "Accessibility" then walk through the rest and report a warning if any of the subsequent tokens don't match their respective parts of the accepted strings.

Just a thought, anyway.

**Comment by man-we** [comment]
Thank you very much for the quick replies, it hadn't occured to me that someone would use their own set of conformance statements. Considering this, looking for "EPUB" and "Accessibility" in the conformance statment and then raising a notice or warning if the rest of the string does not match the specification seems sensible to me.
